src/plugins/piral-breadcrumbs/README.md

```diff
@@ -73,6 +73,20 @@ export function setup(piral: PiletApi) {
 }
 ```
 
+### Rendering Breadcrumbs
+
+Within pilets, you typically _cannot_ access the `Breadcrumbs` component from the `piral-breadcrumbs`. While there are several options to work around this, the simplest is to use a built-in extension provided by Piral called `piral-breadcrumbs`:
```

Comment:
Well, one can always "access" it, but one should never import / include an app shell plugin or core package (such as piral-core etc.) in a pilet. Therefore, I wish we could write this a bit more generic (you should not import ...) and advise to use the `piral-breadcrumbs` extension component.